### ISSUE TITLE
Platform search filters out vapor/jwt

### ISSUE THREAD

**Issue by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#1512]
Adding `platform:linux` to the `jwt` search term filters out `vapor/jwt` even though it supports linux:

<img width="650" alt="CleanShot 2022-01-27 at 11 08 35@2x" src="https://user-images.githubusercontent.com/65520/151337854-90926200-8bd6-4fa3-a9db-3bf3ceda8de0.png">

<img width="660" alt="CleanShot 2022-01-27 at 11 09 01@2x" src="https://user-images.githubusercontent.com/65520/151337866-8b2f6809-2382-448c-9e3e-3324d3879123.png">

<img width="664" alt="CleanShot 2022-01-27 at 11 10 33@2x" src="https://user-images.githubusercontent.com/65520/151338009-4dad76bc-ee74-47da-ba86-46f0e426a466.png">

**Comment by finestructure** [comment]
@0xTim via https://twitter.com/0xTim/status/1486637345927487488

**Comment by finestructure** [comment]
Well, that's embarrassing. The reason it's filtered out is because `packages.platform_compatibility` is `{}` for `vapor/jwt` and the reason for _that_ appears to be simply that I forgot to back-populate the prod db after merging `Package.platformCompatibility`: https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/1429#issuecomment-990950882 😬

**Comment by finestructure** [comment]
Ok, this is now fixed.

However, I _did_ find some evidence in Postico that I ran the prod update on Dec 13, at least partially. It's possibly that I got distracted and didn't finish it (which would explain the tasks not being ticked off, which I def would have, given my love of ticking off tasks...).

The other possibility is that our updating mechanism has a gap and can lead to a disconnect between what's in the build table and what gets aggregated up into `Package.platformCompatibility`. But since we update packages on receiving build reports at the same time we add builds to the builds table that's very unlikely.

Thanks for spotting this, Tim!
